README.md

```diff
@@ -704,14 +701,44 @@ data_labeler = dp.DataLabeler(labeler_type='structured', trainable=True)
 #       data to be ingested with two columns [X, y] where X is the samples and 
 #       y is the labels.
 model_results = data_labeler.fit(x=data['samples'], y=data['labels'], 
-    validation_split=0.2, epochs=2, labels=labels)
+                                 validation_split=0.2, epochs=2, labels=labels)
 
 # final_results, final_confidences are a list of results for each epoch
 epoch_id = 0
 final_results = model_results[epoch_id]["pred"]
 final_confidences = model_results[epoch_id]["conf"]
 ```
 
+The following illustrates how to extend the labels:
+```python
+import dataprofiler as dp
+
+new_labels = ['label1', 'label2', ...]
+data = dp.Data("your_file.csv")  # contains data with new labels
+
+# load default structured Data Labeler w/ trainable set to True
+data_labeler = dp.DataLabeler(labeler_type='structured', trainable=True)
+
+# this will maintain current labels and model weights, but extend the model's 
+# labels
+for label in new_labels:
+    data_labeler.add_label(label)
+
+# For a trainable model, the user must then train the model to be able to 
+# continue using the labeler since the model's graph has likely changed
+# NOTE: data must be in an acceptable format for the preprocessor to interpret.
+#       please refer to the preprocessor/model for the expected data format.
+#       Currently, the DataLabeler cannot take in Tabular data, but requires 
+#       data to be ingested with two columns [X, y] where X is the samples and 
+#       y is the labels.
+model_results = data_labeler.fit(x=data['samples'], y=data['labels'], 
+                                 validation_split=0.2, epochs=2)
+
+# final_results, final_confidences are a list of results for each epoch
+epoch_id = 0
+final_results = model_results[epoch_id]["pred"]
+final_confidences = model_results[epoch_id]["conf"]
```

Comment:
Do we need these last several lines to show adding labels?

Reply:
I think it's good to show a complete story, but if the desire is to remove, i'm okay with that.